### PR TITLE
remove old logs

### PR DESCRIPTION
**Pull request by jimmy-holzer-box**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jimmy-holzer-box has signed the CLA. Thanks for the pull request!

File: box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java [hunk 1]
```diff
@@ -436,29 +434,19 @@ protected synchronized void startAuthenticateUI(BoxSession session) {
...
-        FutureTask<BoxAuthenticationInfo> task = new FutureTask<BoxAuthenticationInfo>(new Callable<BoxAuthenticationInfo>() {
+        FutureTask<BoxAuthenticationInfo> task = new FutureTask<>(new Callable<BoxAuthenticationInfo>() {
             @Override
             public BoxAuthenticationInfo call() throws Exception {
-                BoxAuthenticationInfo refreshInfo = null;
+                BoxAuthenticationInfo refreshInfo;
                 if (session.getRefreshProvider() != null) {
-                    try {
                         refreshInfo = session.getRefreshProvider().refreshAuthenticationInfo(info);
-                    } catch (BoxException e) {
```
Comment (doncung) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java]: Why is this code being removed?

File: box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java [hunk 2]
```diff
@@ -558,16 +544,15 @@ public BoxAuthenticationInfo call() throws Exception {
...
 
         /**
          * This method should return a refreshed authentication info object given one that is expired or nearly expired.
          *
          * @param info the expired authentication information.
          * @return a refreshed BoxAuthenticationInfo object. The object must include the valid access token.
-         * @throws BoxException Exception that should be thrown if there was a problem fetching the information.
          */
-        public BoxAuthenticationInfo refreshAuthenticationInfo(BoxAuthenticationInfo info) throws BoxException;
+        BoxAuthenticationInfo refreshAuthenticationInfo(BoxAuthenticationInfo info);
```
Comment (doncung) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java]: Why are we removing the throws?

File: box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java [hunk 3]
```diff
@@ -925,12 +912,9 @@ protected String getLastAuthentictedUserId(Context context) {
      * @param context current context
      * @return true if an official box application that supports third party authentication is installed.
      */
-    public static boolean isBoxAuthAppAvailable(final Context context) {
+    private static boolean isBoxAuthAppAvailable(final Context context) {
```
Comment (doncung) [inline box-content-sdk/src/main/java/com/box/androidsdk/content/auth/BoxAuthentication.java]: why are you changing this to private?

**Pull request merged**